From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mark Salyzyn <salyzyn@google.com>
Date: Mon, 26 Aug 2019 07:56:13 -0700
Subject: ANDROID: gki_defconfig: set CONFIG_NR_CPUS=32

Consensus is that CONFIG_NR_CPUS of 32 will deal with the future
products with a moderate engineering margin.

Signed-off-by: Mark Salyzyn <salyzyn@google.com>
Test: confirm value propagates to .config
Bug: 139693734
Bug: 139406736
Bug: 139692860
Change-Id: I9687d37da254a612947398a45ae56ab01e676562
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index b70d608b1ee6..2773a84fd939 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -37,6 +37,7 @@ CONFIG_SLAB_FREELIST_RANDOM=y
 CONFIG_SLAB_FREELIST_HARDENED=y
 CONFIG_PROFILING=y
 CONFIG_SCHED_MC=y
+CONFIG_NR_CPUS=32
 CONFIG_SECCOMP=y
 CONFIG_PARAVIRT=y
 CONFIG_COMPAT=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index ae3d165a79c3..a1b2121bde8b 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -35,6 +35,7 @@ CONFIG_SLAB_FREELIST_RANDOM=y
 CONFIG_SLAB_FREELIST_HARDENED=y
 CONFIG_PROFILING=y
 CONFIG_SMP=y
+CONFIG_NR_CPUS=32
 CONFIG_PM_WAKELOCKS=y
 CONFIG_PM_WAKELOCKS_LIMIT=0
 # CONFIG_PM_WAKELOCKS_GC is not set
